airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1618,6 +1634,14 @@ components:
           type: array
           items:
             type: string
+    # Health
+    HealthCheckRead:
+      type: object
+      required:
+        - db
```

Comment:
small thing but just `db` is vague, maybe `db_healthy`

Reply:
this is how we did it for unblocker. why the change of heart? will it be clearer when we add other attributes or less clear?

Reply:
probably more clear -- I'm good with it either way but it saves the 1 second of mental mapping from `db` to `db_healthy` in my head